simphony/cuds/materials.py

```diff
@@ -0,0 +1,99 @@
+from simphony.cuds.material import Material
+
+
+class Materials(object):
+    """A class that contains materials.
+
+    Class manages the materials.
+
+    """
+    def __init__(self, uid=None, data=None, description=""):
```

Comment:
@tuopuu , Sorry. I was looking at 'material.py' instead of 'materials.py'.  Cut and paste error :) Will fix it.